tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "noImplicitThis": true,
     "noImplicitAny": false,
     "strictNullChecks": true,
+    "noUnusedLocals": true,
```

Comment:
I just want to point out that (i think) this won't allow to have unused locals on dev builds (e.g. `yarn start`), is possible to only have them in `yarn build` but not on `yarn start`? 
I'm going to test this later and confirm.

Reply:
I don't think it is, but to be honest I don't like the idea of having compilation targets having different outputs/statuses.

Reply:
From what i remember, you could include one file into the other.
The idea is to make it easier to test changes without having to have a flawless code.
e.g. commenting a path to quickly debug something without the "unused variable" error that does not let you  proceed.

Reply:
Oh and you cal set them to "WARNS" instead of "ERRORS", so they don't fail but you still get the info that your code is "dirty".

Reply:
You can, however warns are supposed to break CI :(
My opinion on this matter is that the more strict the compiler is, the earlier you catch problems that will take a little more time later.